.github/workflows/release-test-version.yml

```diff
@@ -0,0 +1,132 @@
+name: 'Manual Release Test Version'
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Specify a version for the test release (ex. test-browser)"
+        required: true
+        type: string
```

Comment:
Do you know if the input string is trimmed by default? Just to make sure there won't be version ` `

Reply:
I don't think it's trimmed since the default value if not set like we do we would be `''`, feels like a small edge case not too important because the most common case of forgetting to set it would be that the default applies of `test-version`